client/landing/site.landing/coffee/core/utils.coffee

```diff
@@ -2,6 +2,8 @@ $       = require 'jquery'
 kd      = require 'kd'
 kookies = require 'kookies'
 
+RECATCHA_JS = 'https://www.google.com/recaptcha/api.js?onload=onRecaptchaLoaded&render=explicit'
```

Comment:
Good catch! I created `utils.loadRecaptchaScript()` to use it in all views with recaptcha but missed old signup 😞